bigflow/dataflow.py

```diff
@@ -0,0 +1,42 @@
+import typing
+
+from apache_beam import Pipeline
+from apache_beam.options.pipeline_options import PipelineOptions, GoogleCloudOptions
+
+from .workflow import Job, JobContext
+
+
+class BeamJob(Job):
+    def __init__(
+            self,
+            id: str,
+            driver_callable: typing.Callable[[Pipeline, JobContext, typing.Dict], None],
```

Comment:
This function is used as `f(pipeline, context, **kwargs)`, not as `f(pipeline, context, kwargs)`
AFAIK in python 3.7 there is no way to express such callback type (with **kwargs)

Reply:
switched from kwargs to a dict argument